helmfile/charts/networkpolicy-workload/templates/monitoring-np.yaml

```diff
@@ -0,0 +1,229 @@
+{{ if .Values.monitoring.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: node-exporter
+  namespace: monitoring
+spec:
+  podSelector:
+    matchLabels:
+      app: prometheus-node-exporter
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              app: prometheus
+      ports:
+        - port: 9100
+  egress: []
+---
+apiVersion: networking.k8s.io/v1
```

Comment:
In my cluster i still see traffic being blocked for prometheus-backbox-exporter in wc.

Reply:
![image](https://user-images.githubusercontent.com/12862587/192797279-664bd824-ea2a-483c-a426-f8fb7b174bbf.png)

Reply:
Should be fixed now